src/main/java/microsoft/exchange/webservices/data/dns/DnsClient.java

```diff
@@ -43,6 +43,23 @@
 public class DnsClient {
 
   /**
+   * Set up the environment used to construct the DirContext.
+   *
+   * @param dnsServerAddress
+   * @return
+   */
+  static Hashtable<String, String> getEnv(String dnsServerAddress) {
```

Comment:
Isn't that the point of package private? Marking the methods private fails to compile for me, but perhaps I'm missing something.